rdt-open-reader/src/test/java/io/ona/rdt/robolectric/widget/WidgetFactoryRobolectricTest.java

```diff
@@ -33,19 +33,27 @@ public void tearDown() {
     }
 
     public static RDTJsonFormActivity getRDTJsonFormActivity() throws JSONException {
+        return getRDTJsonFormActivity(new JSONObject());
+    }
+
+    public static RDTJsonFormActivity getRDTJsonFormActivity(JSONObject stepObject) throws JSONException {
         return Mockito.spy(Robolectric.buildActivity(RDTJsonFormActivity.class,
-                getJsonFormActivityIntent())
+                getJsonFormActivityIntent(stepObject))
                 .create()
                 .resume()
                 .get());
     }
 
-    private static Intent getJsonFormActivityIntent() throws JSONException {
+    private static Intent getJsonFormActivityIntent(JSONObject stepObject) throws JSONException {
```

Comment:
Could we revert this change, don't think we need the extra argument.

Reply:
Actually, i need a `stepObject` and i can't call the `getStepObject()` method directly in this method that's why I have passed it as a parameter.